R/user.R

```diff
@@ -72,3 +72,47 @@ resetPassword <- function (email) {
             url_base=absoluteURL("../password/change/${token}/", app)
         ))))
 }
+
+#' @rdname catalog-extract
+#' @export
+setMethod("[", c("UserCatalog", "character"), function (x, i, ...) {
+    dots <- list(x=x, i=i, secondary = emails(x))
+    dots <- modifyList(dots, list(...))
+    do.call("callNextMethod", dots)
+})
+
+#' @rdname catalog-extract
+#' @export
+setMethod("[[", c("UserCatalog", "character"), function (x, i, ...) {
+    dots <- list(x=x, i=i, secondary = emails(x))
+    dots <- modifyList(dots, list(...))
+    do.call("callNextMethod", dots)
+})
+
+#' Expropriate all Crunch objects from a user
+#' 
+#' If you want to transfer all teams, projects, and datasets owned by one user 
```

Comment:
Can you add a note of why you'd do such a thing? (e.g. when someone leaves the organization, transfer their things before deleting their account).